backend/nfconfig/config.go

```diff
@@ -296,7 +334,146 @@ func extractGnbNames(slice configmodels.Slice) []string {
+		return nil, false
+	}
+	pccRules := buildSlicePccRules(slice)
+	policyControl := nfConfigApi.NewPolicyControl(*plmn, snssai, pccRules)
+
+	return policyControl, true
+}
+
+func buildSlicePccRules(slice configmodels.Slice) []nfConfigApi.PccRule {
+	/* Implementation assumes that the validation of a Network Slice configuration is done upon group creation/modification.
+	At the time of implementing this, validation is not done, but planned.
+
+	TODO: Remove this comment once Device Group validation is implemented.
+	*/
+	pccRules := []nfConfigApi.PccRule{}
+
+	for _, ruleConfig := range slice.ApplicationFilteringRules {
+		ruleId := ruleConfig.RuleName
+		flows := buildPccFlows(ruleConfig)
+		qos := buildPccQos(ruleConfig)
+		precedence := ruleConfig.Priority
+
+		pccRule := nfConfigApi.NewPccRule(ruleId, flows, qos, precedence)
+		pccRules = append(pccRules, *pccRule)
+	}
+
+	// If slice has no PCC rules, add a default one
+	if len(pccRules) == 0 {
+		pccRules = append(pccRules, *defaultPccRule)
+	}
+
+	return pccRules
+}
+
+func buildPccFlows(ruleConfig configmodels.SliceApplicationFilteringRules) []nfConfigApi.PccFlow {
+	pccFlows := []nfConfigApi.PccFlow{}
+
+	description := buildFlowDescription(ruleConfig)
+	var status nfConfigApi.Status
+	if ruleConfig.Action == "deny" {
+		status = nfConfigApi.STATUS_DISABLED
+	} else {
+		status = nfConfigApi.STATUS_ENABLED
+	}
+
+	flowInfo := nfConfigApi.NewPccFlow(
+		description,
+		nfConfigApi.DIRECTION_BIDIRECTIONAL,
+		status,
+	)
+
+	pccFlows = append(pccFlows, *flowInfo)
+	return pccFlows
+}
+
+func buildFlowDescription(ruleConfig configmodels.SliceApplicationFilteringRules) string {
+	endp := ruleConfig.Endpoint
+	if strings.HasPrefix(endp, "0.0.0.0") {
+		endp = "any"
+	}
+
+	switch ruleConfig.Protocol {
+	case tcp:
+		return buildDescription("tcp", endp, ruleConfig.StartPort, ruleConfig.EndPort)
+	case udp:
+		return buildDescription("udp", endp, ruleConfig.StartPort, ruleConfig.EndPort)
+	default:
+		return fmt.Sprintf("permit out ip from %s to assigned", endp)
+	}
+}
+
+func buildDescription(protocol, endpoint string, startPort, endPort int32) string {
+	if startPort == 0 && endPort == 0 {
+		return fmt.Sprintf("permit out %s from %s to assigned", protocol, endpoint)
+	} else if factory.WebUIConfig.Configuration.SdfComp {
+		return fmt.Sprintf("permit out %s from %s %s-%s to assigned", protocol, endpoint, strconv.FormatInt(int64(startPort), 10), strconv.FormatInt(int64(endPort), 10))
+	} else {
+		return fmt.Sprintf("permit out %s from %s to assigned %s-%s", protocol, endpoint, strconv.FormatInt(int64(startPort), 10), strconv.FormatInt(int64(endPort), 10))
+	}
+}
+
+func buildPccQos(ruleConfig configmodels.SliceApplicationFilteringRules) nfConfigApi.PccQos {
+	pccQos := nfConfigApi.NewPccQos(
+		ruleConfig.TrafficClass.Qci,
+		configapi.ConvertToString(uint64(ruleConfig.AppMbrUplink)),
+		configapi.ConvertToString(uint64(ruleConfig.AppMbrDownlink)),
```

Comment:
I think you need to take into account `BitrateUnit`. That is, these should be like:
```suggestion
		strconv.FormatInt(int64(ruleConfig.AppMbrUplink), 10) + ruleConfig.BitrateUnit,
		strconv.FormatInt(int64(ruleConfig.AppMbrDownlink), 10) + ruleConfig.BitrateUnit,
```

Reply:
Hello @gab-arrobo,
The values in the DB are actually always in `bps`. POST helpers convert values given by the user to `bps`. It's true that the `BitrateUnit` can indicate different units, but this is actually a bug - bitrates are converted to bps before saving to the DB, but the unit given by the user is preserved. As a result, if you POST a DevGroup with Uplink bitrate set to 200 Mbps, GET will return 200000000 Mbps. 
This bug should be addressed separately. Until it's fixed, I think it's safer to assume the value is in `bps` and ignore the `BitrateUnit` field.